### DISCUSSION TITLE
refreshInterval waiting for response

### DISCUSSION THREAD

**Discussion by mjanssen** [vercel/swr#573] [General]
Hey there 👋,

We are using SWR in our dashboards to fetch and show up-to-date data. SWR has helped us a lot throughout building the application, but when our back-end seems to have hickups, SWR keeps sending a stream of requests, without ever getting response.

Is there a way to make SWR wait for the response, before requesting new data with refreshInterval?

Thanks!!

**Comment by piero-la-lune** [comment]
Same issue here, have you found a solution?

Thanks!
